packages/dashboard-core-plugins/src/linker/Linker.tsx

```diff
@@ -354,7 +353,7 @@ export class Linker extends Component<LinkerProps, LinkerState> {
           'endPanel no longer exists, ignoring unsetFilterValue',
           panelId
         );
-      } else if (isLinkablePanel(endPanel)) {
+      } else if (isLinkablePanel(endPanel) && columnType != null) {
```

Comment:
`unsetFilterValue` should just allow null columnType instead. By checking for `null` here you're changing one of the code paths.
```suggestion
      } else if (isLinkablePanel(endPanel)) {
```